app/presenters/organisations/show_presenter.rb

```diff
@@ -120,6 +122,34 @@ def featured_policies
       }
     end
 
+    def has_foi_contacts?
+      true if @org.foi_contacts && @org.foi_contacts.length.positive?
+    end
+
+    def foi_contacts
+      @org.foi_contacts.map do |foi_contact|
+        data = {
+          post_addresses: foi_contact["details"]["post_addresses"].map { |post|
```

Comment:
You're mixing use of `map do...end` and `map {}` here - we seem to prefer `do...end` in general, so probably makes it more readable if we always do that.

Reply:
I tried `do...end` but it didn't work, I think because it's inside an attribute declaration? Unless I'm misunderstanding. Can you show what you mean?

Reply:
The following works for me (I think):

```
data = {
    title: foi_title(foi_contact["details"]["title"]),
    post_addresses: foi_contact["details"]["post_addresses"].map do |post|
        data = ""
        .....etc
    end,
    email_addresses: foi_contact["details"]["email_addresses"].map do |email|
        make_email_link(email["email"])
    end,
    description: foi_description(foi_contact["details"]["description"])
}
```